altapay.php

```diff
@@ -364,7 +363,7 @@ function altapayCaptureCallback() {
 
 			try {
 				$api = new CaptureReservation( $settings->getAuth() );
-				$api->setAmount( (float) number_format( $amount, 2, '.', '' ) );
+				$api->setAmount( (float) round( $amount, 2 ) );
```

Comment:
Redundant cast since `round` returns float.
```suggestion
				$api->setAmount( round( $amount, 2 ) );
```